libclc/CMakeLists.txt

```diff
@@ -394,6 +401,7 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
         DEPENDS ${builtins_link_lib}
       )
       add_custom_target( "prepare-${spv_suffix}" ALL DEPENDS "${spv_suffix}" )
+      set_target_properties( "prepare-${spv_suffix}" PROPERTIES FOLDER "libclc/Sourcegenning" )
```

Comment:
I wonder whether it is confusing to have the SPIR-V `prepare-xxx` target labelled `libclc/Sourcegenning` but the non-SPIR-V `prepare-xxx` target labeled `libclc/Device IR/Prepare`. Both being called `prepare` makes me think they should be grouped similarly.

Note that I don't currently have any way of testing out this patch as I haven't got VS Code or XCode available to me right now. It might be that the dichotomy as currently in this patch makes sense.